StaticHelpers/PhoneBookProcesser.cs

```diff
@@ -0,0 +1,30 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+
+namespace PhoneBook.StaticHelpers
+{
+    class PhoneBookProcesser
+    {
+        public static void GetPhoneBookFromFile(string path)
+        {
+            bool isNumberValid;
+            List<string> numberErrors = new List<string>();
+            List<string> separatorErrors = new List<string>();
+            Console.WriteLine("PhoneBook Program:");
+            string allText = FileHelpers.TextFromFileReader(path);
+            Console.WriteLine(allText);
+            List<Person> persons = Person.TextToPersonInfo(allText, separatorErrors);
+            for (int index = 0; index < persons.Count; index++)
+            {
+                isNumberValid = true;
+                if (!persons[index].Number.Validator())
+                    isNumberValid = false;
+                numberErrors.Add(isNumberValid ? null : StringHelpers.invalidNumberMessage);
```

Comment:
no need to add null in array

Reply:
In my logic i need to know info about every row. Is everythig ok with number in row or no. If it is ok i am adding null.

Reply:
Please rewrite that logic and store each line record with message. You can use `Dictionary` to reach that